src/index.html

```diff
@@ -1,13 +1,336 @@
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">Bring joy back <br> to the everyday</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="container scroll-disable">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+              src="images/logo.svg"
+              alt="Nothing logo"
+            >
+          </a>
+          <div class="header__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#recommended">Recommended</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Categories</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about">About</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contact">Contact</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+          <a href="tel:+1 234 5555-55-55" class="menu__call-to-order">
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main-page">
+      <div class="container">
+        <div class="main__content">
+          <section class="recommended" id="recommended">
+
+            <h2 class="section-title">Recommended</h2>
+
+            <div class="recommended__products">
+              <article class="product recommended__product">
+
+                <img
+                  src="./images/products/phone(1).png"
+                  alt="phone(1)"
+                  class="product__photo"
+                >
+
+                <h3 class="product__title">Phone (1)</h3>
+
+                <p
+                  class="product__description"
+                >
+                Designed with instinct, to bring joy back to the everyday. Through the Glyph Interface, a perfected OS and exceptional dual camera. All startlingly fast.
+                </p>
+
+                <p class="product__price">$ 499.00</p>
+
+              </article>
+              <article class="product recommended__product">
+
+                <img
+                  src="./images/products/ear(2).png"
+                  alt="ear(2)"
+                  class="product__photo"
+                >
+
+                <h3 class="product__title">Ear (2)</h3>
+
+                <p
+                  class="product__description"
+                >
```

Comment:
Need to check and fix it everywhere
```suggestion
                <p class="product__description">
```